mm2src/komodefi_cli/Cargo.lock

```diff
@@ -132,7 +132,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "061a7acccaa286c011ddc30970520b98fa40e00c9d644633fb26b5fc63a265e3"
 dependencies = [
  "proc-macro2",
- "quote 1.0.27",
+ "quote 1.0.33",
```

Comment:
Please try to use same version that is used by mm2

Reply:
done. had to do some refactoring for `skip_serializing_none` crate

Reply:
No problem, we can leave it as is then.